src/pages/customer/Cards.tsx

```
@@ -196,8 +210,30 @@ const CustomerCards = () => {
       // First synchronize enrollments to cards to ensure all enrolled programs have cards
       await syncEnrollments();
       
+      // Additional sync for new users - ensure all enrollments have cards
+      try {
+        const syncResult = await LoyaltyCardService.syncEnrollmentsToCards(String(actualCustomerId));
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)